client/apps/articles_list/components/articles_list.jsx

```diff
@@ -142,7 +144,8 @@ export class ArticlesList extends Component {
 
 const mapStateToProps = (state) => ({
   channel: state.app.channel,
-  apiURL: state.app.apiURL
+  apiURL: state.app.apiURL,
+  user: state.app.user
```

Comment:
that line is the fix btw, the `user` object was not being passed into props

Reply:
Cool!  Thanks for adding a test for this... ready to go but you have to pass Peril first.

Reply:
Also do you mind taking care of the styled-components TODO? Should be a fast fix.